packages/fetchai/connections/ledger/connection.yaml

```diff
@@ -22,7 +22,7 @@ config:
   ledger_apis:
     ethereum:
       address: http://127.0.0.1:8545
-      gas_price: 50
```

Comment:
So we don't allow for constant gas price anymore. For 95% per cent of the cases, the new approach is better, but sometimes it could be useful to hard code the gas price.

Reply:
I think this hard-coded approach is quite dangerous. It's safer to make transaction related values be specified explicitly and fail otherwise.

Reply:
Although, we could explicitly offer it as a strategy.